pkg/layer/builder.go

```diff
@@ -71,3 +72,8 @@ func (b *Builder) Infobox(infobox *Infobox) *Builder {
 	b.base.infobox = infobox
 	return b
 }
+
+func (b *Builder) Tags(tags tag.List) *Builder {
+	b.base.tags = tags
+	return b
```

Comment:
NewTagList returns `*tag.List` but this func accepsts `tag.List`, which does not make sense.

```suggestion
func (b *Builder) Tags(tags *tag.List) *Builder {
	b.base.tags = tags.Clone()
	return b
```

Define Copy method to Tag